Jint/Runtime/Interop/ClrHelper.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Jint.Native;
+
+namespace Jint.Runtime.Interop
```

Comment:
for new files, can also use file-scoped namespaces, at some point when there's not so many pending PRs a mass change will be done towards file-scoped namespaces